tensorflow_addons/utils/keras_utils.py

```diff
@@ -14,6 +14,20 @@
 # ==============================================================================
 """Utilities for tf.keras."""
 
+from tensorflow.keras import backend
```

Comment:
Access through `tf.keras.backend.image_data_format()` so we don't need to access private API.

Reply:
Thst's called in the `normalize_data_format()` method. Should I still add that to the classes?